contrib/wireshark/ucx_le64.lua

```diff
@@ -279,33 +274,33 @@ local amo_sw_opcode = {
     [5] = "CSWAP",
 }
 
-local ucp_amo_sw_proto = Proto("UCX_AMO_SW", "UCX Atomic SW")
+local ucp_sw_amo_proto = Proto("UCX_UCP_SW_AMO", "UCX UCP SW Atomic")
```

Comment:
minor: I'd avoid using names like `UCX_UCP...`, better to use just `UCP_...`

Reply:
fixed for sw amo and ucp address, retested for both.